docs/Deploying.md

```diff
@@ -44,25 +60,25 @@ After doing this, you can look in the tomcat log file (`$CATALINA_HOME/logs/cata
 
     INFO: Deployment of web application archive /Users/ecerami/libraries/apache-tomcat-7.0.59/webapps/cbioportal.war has finished in 13,009 ms
 
-# Verify the Web Application
+## Verify the Web Application
 
 Lastly, open a browser and go to:  
 
 [http://localhost:8080/cbioportal/](http://localhost:8080/cbioportal/)
 
-# Important
+## Important
 
 Each time you add new data or modify any code, you must redeploy the WAR file.
```

Comment:
I like Pieter option since it is much clear to understand. However, I would rephrase the first point:

- "Each time you modify any java code, you must recompile and redeploy the WAR file."